### DISCUSSION TITLE
copy only stream rtsp

### DISCUSSION THREAD

**Discussion by man8531** [Motion-Project/motion#1764] [General]
Hello
Is it possible to just copy the rtsp stream and disable any stream processing so as not to load the cpu? I don't need image processing, I only need to copy streams from cameras. my configurations
[motion.conf.txt](https://github.com/Motion-Project/motion/files/14928608/motion.conf.txt)
[cam.conf.txt](https://github.com/Motion-Project/motion/files/14928607/cam.conf.txt)
I do not speak english. google translate

**Comment by tosiara** [comment]
You can disable motion detection, turn on emulate_motion and movie_passthrough. But in that case motion will still need to decode rtsp frames which will take a lot of CPU. If you just need to copy the stream, you can use plain ffmpeg:

`ffmpeg -i rtsp://url -vcodec copy -an copy.mp4`
